src/improvements/tasks/eth/008-deputy-pause-module-activation/README.md

```diff
@@ -0,0 +1,28 @@
+# DeputyPauseModule Installation - OP Mainnet
+
```

Comment:
I think you'll probably want to use 010-deputy-pause-module-activation given we have u15 runbooks ready to be merged at 008 and 009.

Reply:
Thank you will update this PR and the other to 10 and 11 then!